deepspeed/runtime/engine.py

```diff
@@ -78,6 +78,8 @@
 from deepspeed.profiling.flops_profiler.profiler import FlopsProfiler
 from deepspeed.utils.logging import print_json_dist
 
+from deepspeed.inference.config import DtypeEnum
```

Comment:
We should probably move the `DTypeEnum` to `deepspeed/runtime/config_utils.py` or `deepspeed/runtime/config.py` if it's being used in multiple configs. We can do that in this PR or a future PR when we refactor the main runtime config to use pydantic.